pkg/stanza/fileconsumer/internal/reader/reader_test.go

```diff
@@ -193,7 +193,7 @@ func TestFlushPeriodEOF(t *testing.T) {
 	// Create a long enough initial token, so the scanner can't read the whole file at once
 	aContentLength := 2 * 16 * 1024
 	content := []byte(strings.Repeat("a", aContentLength))
-	content = append(content, '\n', 'b')
+	content = append(content, '\n', 'b', '\n')
```

Comment:
I think this change might be invalidating the premise of the test. By adding an additional newline, the second token would just be emitted immediately, rather than hanging until there is a flush.

Reply:
Yeap, I think this is true. I was only focusing on ensuring that the first token is not split. 

So it seems that on windows we don't manage this "hanging" situation properly.